[Bug]: test_worker_node_restart_during_pvc_expansion failed with error "DeploymentConfig is deprecated"

### Description
Failed test case tests/functional/pv/pvc_resize/test_node_restart_during_pvc_expansion.py::TestNodeRestartDuringPvcExpansion::test_worker_node_restart_during_pvc_expansion

Error:
```
          raise CommandFailed(
                f"Error during execution of command: {masked_cmd}."
                f"\nError is {masked_stderr}"
            )
E               ocs_ci.ocs.exceptions.CommandFailed: Error during execution of command: oc --kubeconfig /home/jenkins/current-cluster-dir/openshift-cluster-dir/auth/kubeconfig -n namespace-test-66afb617d30148c698355ef81 create -f /tmp/DeploymentConfiggmirhrl6 -o yaml.
E               Error is Warning: apps.openshift.io/v1 DeploymentConfig is deprecated in v4.14+, unavailable in v4.10000+
E               Error from server (BadRequest): error when creating "/tmp/DeploymentConfiggmirhrl6": DeploymentConfig in version "v1" cannot be handled as a DeploymentConfig: json: cannot unmarshal object into Go struct field DeploymentConfigSpec.spec.selector of type string



ocs_ci/utility/utils.py:737: CommandFailed
```


### Steps to Reproduce
1.Run tests/functional/pv/pvc_resize/test_node_restart_during_pvc_expansion.py::TestNodeRestartDuringPvcExpansion::test_worker_node_restart_during_pvc_expansion

### Actual Behavior
Using deprecated deploymenconfig
### Expected Behavior
Use deployment
### Impact (likelihood of reproduction, impact on the cluster and on other tests, etc.)
Test cases that use deploymentconfig
### Screenshots (if applicable)

### Environment
- Test Suite(s): tier4b
- Platform(s): AWS, any other
- Version(s): 4.21
- OS:

### Additional Context

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -7059,7 +7059,7 @@ def create_pvcs_and_pods(multi_pvc_factory, pod_factory, service_account_factory
                 interface = constants.CEPHBLOCKPOOL
 
             if deployment:
-                pod_dict_path = pod_dict_path or constants.FEDORA_DC_YAML
+                pod_dict_path = pod_dict_path or constants.FEDORA_DEPLOY_YAML
             elif pvc_obj.volume_mode == "Block":
                 pod_dict_path = pod_dict_path or constants.CSI_RBD_RAW_BLOCK_POD_YAML
             else:

